src/main/java/seedu/address/logic/commands/FindPolicyCommand.java

```diff
@@ -0,0 +1,42 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.policy.PolicyNameContainsKeywordsPredicate;
+
+/**
+ * Finds and lists all policies in address book whose name contains any of the argument keywords.
+ * Keyword matching is case insensitive.
+ */
+public class FindPolicyCommand extends Command {
```

Comment:
Bug: Using `findpolicy` while the list is showing people still shows list of people.

Reply:
Will work on this, thank you! Seems to be a bug with `find` as well: using `find` while list is showing policies still shows list of policies.